{
    "repository": "mysto/python-fpe",
    "clone_url": "https://github.com/mysto/python-fpe.git",
    "timestamp": "2023-05-29T15:59:51.860438Z",
    "clone_success": true,
    "number of actions": 1,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_stdout": "[Python FPE CI/build] \ud83d\ude80  Start image=crawlergpt:latest\n[Python FPE CI/build]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[Python FPE CI/build]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Python FPE CI/build]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Python FPE CI/build]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /tmp/de65f406-fe28-11ed-a890-af2cc187fc11/mysto-python-fpe] user=0 workdir=\n[Python FPE CI/build]   \u2601  git clone 'https://github.com/actions/setup-python' # ref=v4\n[Python FPE CI/build] \ud83e\uddea  Matrix: map[python-version:3.7]\n[Python FPE CI/build] \u2b50 Run Main actions/checkout@v3\n[Python FPE CI/build]   \u2705  Success - Main actions/checkout@v3\n[Python FPE CI/build] \u2b50 Run Main Set up Python 3.7\n[Python FPE CI/build]   \ud83d\udc33  docker cp src=/home/andre-silva/.cache/act/actions-setup-python@v4/ dst=/var/run/act/actions/actions-setup-python@v4/\n[Python FPE CI/build]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /var/run/act/actions/actions-setup-python@v4/] user=0 workdir=\n[Python FPE CI/build]   \ud83d\udc33  docker exec cmd=[node /var/run/act/actions/actions-setup-python@v4/dist/setup/index.js] user= workdir=\n[Python FPE CI/build]   \ud83d\udcac  ::debug::Python is expected to be installed into /opt/hostedtoolcache\n[Python FPE CI/build]   \u2753  ::group::Installed versions\n[Python FPE CI/build]   \ud83d\udcac  ::debug::Semantic version spec of 3.7 is 3.7\n[Python FPE CI/build]   \ud83d\udcac  ::debug::isExplicit: \n[Python FPE CI/build]   \ud83d\udcac  ::debug::explicit? false\n[Python FPE CI/build]   \ud83d\udcac  ::debug::isExplicit: 2.7.18\n[Python FPE CI/build]   \ud83d\udcac  ::debug::explicit? true\n[Python FPE CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.5.10\n[Python FPE CI/build]   \ud83d\udcac  ::debug::explicit? true\n[Python FPE CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.6.14\n[Python FPE CI/build]   \ud83d\udcac  ::debug::explicit? true\n[Python FPE CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.7.11\n[Python FPE CI/build]   \ud83d\udcac  ::debug::explicit? true\n[Python FPE CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.8.11\n[Python FPE CI/build]   \ud83d\udcac  ::debug::explicit? true\n[Python FPE CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.9.6\n[Python FPE CI/build]   \ud83d\udcac  ::debug::explicit? true\n[Python FPE CI/build]   \ud83d\udcac  ::debug::evaluating 6 versions\n[Python FPE CI/build]   \ud83d\udcac  ::debug::matched: 3.7.11\n[Python FPE CI/build]   \ud83d\udcac  ::debug::checking cache: /opt/hostedtoolcache/Python/3.7.11/x64\n[Python FPE CI/build]   \ud83d\udcac  ::debug::Found tool in cache Python 3.7.11 x64\n[Python FPE CI/build]   | Successfully set up CPython (3.7.11)\n[Python FPE CI/build]   \u2753  ::endgroup::\n[Python FPE CI/build]   \u2753 add-matcher /run/act/actions/actions-setup-python@v4/.github/python.json\n[Python FPE CI/build]   \u2705  Success - Main Set up Python 3.7\n[Python FPE CI/build]   \u2699  ::set-env:: LD_LIBRARY_PATH=/opt/hostedtoolcache/Python/3.7.11/x64/lib\n[Python FPE CI/build]   \u2699  ::set-env:: pythonLocation=/opt/hostedtoolcache/Python/3.7.11/x64\n[Python FPE CI/build]   \u2699  ::set-env:: PKG_CONFIG_PATH=/opt/hostedtoolcache/Python/3.7.11/x64/lib/pkgconfig\n[Python FPE CI/build]   \u2699  ::set-env:: Python_ROOT_DIR=/opt/hostedtoolcache/Python/3.7.11/x64\n[Python FPE CI/build]   \u2699  ::set-env:: Python2_ROOT_DIR=/opt/hostedtoolcache/Python/3.7.11/x64\n[Python FPE CI/build]   \u2699  ::set-env:: Python3_ROOT_DIR=/opt/hostedtoolcache/Python/3.7.11/x64\n[Python FPE CI/build]   \u2699  ::set-output:: python-version=3.7.11\n[Python FPE CI/build]   \u2699  ::set-output:: python-path=/opt/hostedtoolcache/Python/3.7.11/x64/bin/python\n[Python FPE CI/build]   \u2699  ::add-path:: /opt/hostedtoolcache/Python/3.7.11/x64\n[Python FPE CI/build]   \u2699  ::add-path:: /opt/hostedtoolcache/Python/3.7.11/x64/bin\n[Python FPE CI/build] \u2b50 Run Main Install dependencies\n[Python FPE CI/build]   \ud83d\udc33  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/2] user= workdir=\n[Python FPE CI/build]   | Requirement already satisfied: pip in /opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages (21.2.4)\n[Python FPE CI/build]   | Collecting pip\n[Python FPE CI/build]   |   Downloading pip-23.1.2-py3-none-any.whl (2.1 MB)\n[Python FPE CI/build]   | Installing collected packages: pip\n[Python FPE CI/build]   |   Attempting uninstall: pip\n[Python FPE CI/build]   |     Found existing installation: pip 21.2.4\n[Python FPE CI/build]   |     Uninstalling pip-21.2.4:\n[Python FPE CI/build]   |       Successfully uninstalled pip-21.2.4\n[Python FPE CI/build]   | Successfully installed pip-23.1.2\n[Python FPE CI/build]   | Collecting ruff\n[Python FPE CI/build]   |   Downloading ruff-0.0.270-py3-none-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (5.5 MB)\n[Python FPE CI/build]   |      \u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501 5.5/5.5 MB 27.7 MB/s eta 0:00:00\n[Python FPE CI/build]   | Collecting pytest\n[Python FPE CI/build]   |   Downloading pytest-7.3.1-py3-none-any.whl (320 kB)\n[Python FPE CI/build]   |      \u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501 320.5/320.5 kB 7.8 MB/s eta 0:00:00\n[Python FPE CI/build]   | Collecting iniconfig (from pytest)\n[Python FPE CI/build]   |   Downloading iniconfig-2.0.0-py3-none-any.whl (5.9 kB)\n[Python FPE CI/build]   | Collecting packaging (from pytest)\n[Python FPE CI/build]   |   Downloading packaging-23.1-py3-none-any.whl (48 kB)\n[Python FPE CI/build]   |      \u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501 48.9/48.9 kB 5.0 MB/s eta 0:00:00\n[Python FPE CI/build]   | Collecting pluggy<2.0,>=0.12 (from pytest)\n[Python FPE CI/build]   |   Downloading pluggy-1.0.0-py2.py3-none-any.whl (13 kB)\n[Python FPE CI/build]   | Collecting exceptiongroup>=1.0.0rc8 (from pytest)\n[Python FPE CI/build]   |   Downloading exceptiongroup-1.1.1-py3-none-any.whl (14 kB)\n[Python FPE CI/build]   | Collecting tomli>=1.0.0 (from pytest)\n[Python FPE CI/build]   |   Downloading tomli-2.0.1-py3-none-any.whl (12 kB)\n[Python FPE CI/build]   | Collecting importlib-metadata>=0.12 (from pytest)\n[Python FPE CI/build]   |   Downloading importlib_metadata-6.6.0-py3-none-any.whl (22 kB)\n[Python FPE CI/build]   | Collecting zipp>=0.5 (from importlib-metadata>=0.12->pytest)\n[Python FPE CI/build]   |   Downloading zipp-3.15.0-py3-none-any.whl (6.8 kB)\n[Python FPE CI/build]   | Collecting typing-extensions>=3.6.4 (from importlib-metadata>=0.12->pytest)\n[Python FPE CI/build]   |   Downloading typing_extensions-4.6.2-py3-none-any.whl (31 kB)\n[Python FPE CI/build]   | Installing collected packages: zipp, typing-extensions, tomli, ruff, packaging, iniconfig, exceptiongroup, importlib-metadata, pluggy, pytest\n[Python FPE CI/build]   | Successfully installed exceptiongroup-1.1.1 importlib-metadata-6.6.0 iniconfig-2.0.0 packaging-23.1 pluggy-1.0.0 pytest-7.3.1 ruff-0.0.270 tomli-2.0.1 typing-extensions-4.6.2 zipp-3.15.0\n[Python FPE CI/build]   | \n[Python FPE CI/build]   | [notice] A new release of pip is available: 20.1.1 -> 23.1.2\n[Python FPE CI/build]   | [notice] To update, run: pip install --upgrade pip\n[Python FPE CI/build]   | Collecting pycryptodome (from -r requirements.txt (line 1))\n[Python FPE CI/build]   |   Downloading pycryptodome-3.18.0-cp35-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.1 MB)\n[Python FPE CI/build]   |      \u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501\u2501 2.1/2.1 MB 12.7 MB/s eta 0:00:00\n[Python FPE CI/build]   | Installing collected packages: pycryptodome\n[Python FPE CI/build]   | Successfully installed pycryptodome-3.18.0\n[Python FPE CI/build]   | \n[Python FPE CI/build]   | [notice] A new release of pip is available: 20.1.1 -> 23.1.2\n[Python FPE CI/build]   | [notice] To update, run: pip install --upgrade pip\n[Python FPE CI/build]   \u2705  Success - Main Install dependencies\n[Python FPE CI/build] \u2b50 Run Main Lint with ruff\n[Python FPE CI/build]   \ud83d\udc33  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/3] user= workdir=\n[Python FPE CI/build]   \u2705  Success - Main Lint with ruff\n[Python FPE CI/build] \u2b50 Run Main Test with pytest\n[Python FPE CI/build]   \ud83d\udc33  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/4] user= workdir=\n[Python FPE CI/build]   | ============================= test session starts ==============================\n[Python FPE CI/build]   | platform linux -- Python 3.7.11, pytest-7.3.1, pluggy-1.0.0\n[Python FPE CI/build]   | rootdir: /tmp/de65f406-fe28-11ed-a890-af2cc187fc11/mysto-python-fpe\n[Python FPE CI/build]   | collected 13 items\n[Python FPE CI/build]   | \n[Python FPE CI/build]   | ff3/ff3_test.py .............                                            [100%]\n[Python FPE CI/build]   | \n[Python FPE CI/build]   | ============================== 13 passed in 4.74s ==============================\n[Python FPE CI/build]   \u2705  Success - Main Test with pytest\n[Python FPE CI/build] \u2b50 Run Post Set up Python 3.7\n[Python FPE CI/build]   \ud83d\udc33  docker exec cmd=[node /var/run/act/actions/actions-setup-python@v4/dist/cache-save/index.js] user= workdir=\n[Python FPE CI/build]   \u2705  Success - Post Set up Python 3.7\n[Python FPE CI/build] failed to remove container: Delete \"http://%2Fvar%2Frun%2Fdocker.sock/v1.41/containers/bfd5ac9c615a2b89dd6ce0878217e56fd5be76194a0861f2a64ff4dc6ae12717?force=1&v=1\": context deadline exceeded\n[Python FPE CI/build] \ud83c\udfc1  Job succeeded\n",
    "actions_stderr": "Error: Error occurred running finally: Error occurred running finally: context deadline exceeded (original error: <nil>) (original error: <nil>)\n"
}